api/opportunities/models/opportunities.settings.json

```diff
@@ -0,0 +1,56 @@
+{
+  "kind": "collectionType",
+  "collectionName": "opportunities",
+  "info": {
+    "name": "Opportunities",
+    "description": ""
+  },
+  "options": {
+    "increments": true,
+    "timestamps": true,
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "title": {
+      "type": "string",
+      "required": true
+    },
+    "skills": {
+      "collection": "interest"
+    },
+    "level": {
+      "type": "enumeration",
+      "enum": [
+        "Beginner",
+        "Intermediate",
+        "Advanced"
+      ],
+      "required": true
+    },
+    "commitmentHoursPerWeek": {
+      "type": "integer",
+      "required": true
+    },
+    "description": {
+      "type": "richtext",
```

Comment:
Can you remind me what's the difference between string and richtext?

Reply:
String is basic string, it can be a URL or a name but nothing sophisticated. Rich text allows text to be formatted using markdowns like in GitHub comments

Bold **bold**
Italics _Italics_

Etc...

Reply:
Is the frontend going to render the markdown?

Reply:
Yeah the front end of the client app and even Strapi front end will render a markdown editor